source/acquaman/REIXS/REIXSXESScanController.cpp

```diff
@@ -182,6 +182,16 @@ bool REIXSXESScanController::startImplementation() {
 	return true;
 }
 
+void REIXSXESScanController::pauseImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->pauseDwelling();
+}
+
+void REIXSXESScanController::resumeImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->resumeDwelling();
+}
```

Comment:
@iainworkman Good call. When scan is cancelled/skipped, we will stop the timer ans set it to null. But Veto should be off when scan is cancelled/skipped/finished and be on when scan start. I will update that.

@dretrex I didn't test it yet. I want Iain to have a check whether I'm on the right track.